### PR TITLE
Adding fake token warning and replacing `Add Token` labels with `Import Tokens`

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
This PR is a part of issue #11619 
This PR replaces the `Add Tokens` labels with `Import Tokens` and a warning message on the custom token addition page for scam and security risk awareness.

Screen Shot 2021-09-03 at 3 44 22 PM

Screen Shot 2021-09-03 at 3 44 46 PM

Screen Shot 2021-08-09 at 10 15 46 PM

Screen Shot 2021-08-09 at 10 17 30 PM

Screen Shot 2021-08-09 at 10 16 38 PM

### CODE REVIEW

File: ui/pages/import-token/import-token.component.js [hunk 1]
```diff
@@ -266,7 +266,25 @@ class AddToken extends Component {
...
+      <div className="import-token__custom-token-form">
+        <ActionableMessage
+          message={this.context.t('fakeTokenWarning', [
+            <Button
+              type="link"
+              key="import-token-fake-token-warning"
+              className="import-token__link"
+              rel="noopener noreferrer"
+              target="_blank"
+              href=" https://metamask.zendesk.com/hc/en-us/articles/4403988839451"
```
Comment (ryanml) [inline ui/pages/import-token/import-token.component.js:278]: nit, fix leading whitespace in `href`

File: test/e2e/tests/permissions.spec.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ describe('Permissions', function () {
           text: 'Connected sites',
           tag: 'h2',
         });
-
+        await driver.delay(xxLargeDelayMs);
```
Comment (adonesky1) [inline test/e2e/tests/permissions.spec.js:65]: Hmmm why was this change required?

**Review by adonesky1 — COMMENTED** [review]
[nit] seems like you should rename the file and component in confirm-add-token.stories.js as well

File: test/e2e/tests/permissions.spec.js (hunk 2 above)
Comment (NiranjanaBinoy) [reply test/e2e/tests/permissions.spec.js:65]: The test is breaking without that delay

**Review by adonesky1 — DISMISSED** [review]
LGTM!

File: ui/pages/import-token/import-token.stories.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { boolean } from '@storybook/addon-knobs';
+
+import ImportToken from './import-token.component';
+
+export default {
+  title: 'Import Token',
+};
+
+export const ImportTokenComponent = () => {
+  return <ImportToken showSearchTab={boolean('Show Search Tab', false)} />;
```
Comment (darkwing) [inline ui/pages/import-token/import-token.stories.js:11]: This needs to be `Boolean`

Comment (brad-decker) [reply ui/pages/import-token/import-token.stories.js:11]: When did you change your username, Mark? (cc @Gudahtt, lol)

Comment (NiranjanaBinoy) [reply ui/pages/import-token/import-token.stories.js:11]: This seems to be the `boolean` function imported from `@storybook/addon-knobs`.  I haven't made any change in the file other than renaming it.

Comment (brad-decker) [reply ui/pages/import-token/import-token.stories.js:11]: ah! oops.

**Review by brad-decker — DISMISSED** [review]

File: ui/pages/pages.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /** Please import your files in alphabetical order **/
-@import 'add-token/index';
+@import 'import-token/index';
```
Comment (mcmire) [inline ui/pages/pages.scss:2]: Should this be moved down to make this list alphabetical as per the comment above?

**Review by danjm — DISMISSED** [review]
LGTM once merge conflicts are resolved

File: ui/pages/pages.scss (hunk 4 above)
Comment (danjm) [reply ui/pages/pages.scss:2]: I'm okay with not enforcing import ordering that isn't yet enforced by our linter

Comment (brad-decker) [reply ui/pages/pages.scss:2]: Sounds like a challenge. 🥇

**Review by brad-decker — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**